contracts/test/semantics.ts

```diff
@@ -22,11 +22,11 @@ describe('EVM Semantics', () => {
   });
 
   it('eth_call maximum return length vs gas limit', async () => {
-    const i = 1787872;
+    const i = 1211104;
```

Comment:
What's this about?

Reply:
Some time since the tests were made and now the amount of gas it costs has changed.

I haven't been able to track down what exactly, is it different Solidity version, different Sapphire version. We used to be able to pass 1.7mb of calldata, now we can only pass 1.2mb of calldata without it running out of gas in the block.